src/api/index.js

```diff
@@ -153,7 +144,7 @@ export default class Api {
             return Object.defineProperty(obj, prop, {
                 get() {
                     return instance.get();
-                }
+                },
```

Comment:
Dangling commas are recommended by [some style guides](https://github.com/airbnb/javascript#commas--dangling). Please, let me know if you are against dangling comas and I will add a linting rule.